core/src/main/java/io/ultrabrew/metrics/data/DistributionBucketIF.java

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021, Oath Inc.
+// Licensed under the terms of the Apache License 2.0 license. See LICENSE file in Ultrabrew Metrics
+// for terms.
+
+package io.ultrabrew.metrics.data;
+
+/**
+ * A distribution bucket specification used for histograms. A distribution bucket is an sorted array
+ * of unique numeric values. Each bucket represents a distribution range of the measurement.
+ *
+ * @see DistributionBucket
+ * @see DoubleValuedDistributionBucket
+ */
+public interface DistributionBucketIF<B extends DistributionBucketIF> {
+
+  String UNDERFLOW = "underflow";
```

Comment:
Would be good to comment how these are used. Reading the PR with a device that doesnt allow searching to see where this is used and how, but question arises is this for internal book-keeping purpose only? This is in an interface so assumption is that this is part of interface and should be documented what the expected use is.

Reply:
Add the doc. Please, take a look